DigitalLearningSolutions.Web/Controllers/ForgotPasswordController.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers
+{
+    using Microsoft.AspNetCore.Mvc;
+    using DigitalLearningSolutions.Web.ViewModels.ForgotPassword;
+    using DigitalLearningSolutions.Web.ViewModels.LearningSolutions;
+
+    public class ForgotPasswordController : Controller
+    {
+
+        [Route("/ForgotPassword")]
+        public IActionResult ForgotPassword()
+        {
+            var model = new ForgotPasswordViewModel();
+            return View(model);
+        }
+
+        [HttpPost]
+        public IActionResult SubmitEmail(string emailAddress)
```

Comment:
Hmm.. I'd probably name this RecoverPassword as we'll also be doing some db changes and validation here. :)

Also, do you think it'd be worth redirecting to a /ForgotPassword/Confirm url when everything here is successful?